packages/rocketchat-authorization/client/views/permissions.js

```diff
@@ -1,66 +1,121 @@
 /* globals ChatPermissions */
+import {permissionLevel} from '../../lib/rocketchat';
+
+const whereNotSetting = {
```

Comment:
`$ne` is not supported by Minimongo. Thus, had to fallback to `$where`. Still, this does not seem to work, see the `.filter`-comment some lines below